src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -417,8 +418,45 @@ private void androidSdkManager(String[] args) throws IOException, InterruptedExc
      * @throws InterruptedException in case anything goes wrong.
      */
     private void fetchFromSdkManager() throws IOException, InterruptedException {
-        Logger.logDebug("Downloading Android toolchain...");
+        Logger.logInfo("Downloading Android NDK and toolchain...");
         androidSdkManager(ANDROID_SDK_PACKAGES);
-        Logger.logDebug("Done downloading Android toolchain");
+        Logger.logInfo("Android NDK and toolchain downloaded successfully");
+    }
+
+    /**
+     * Prints "." every second the background process is running.
+     * The feedback is helpful for end user. A lack of feedback can lead to an impression that the process is stuck.
+     */
+    private static class ProcessWithFeedback implements Callable<Integer> {
+
+        private final ProcessRunner processRunner;
+        private final String processName;
+        private final Timer timer;
+        
+        static Integer execute(ProcessRunner processRunner, String processName) throws IOException, InterruptedException {
+            return new ProcessWithFeedback(processRunner, processName).call();
+        }
+
+        private ProcessWithFeedback(ProcessRunner processRunner, String processName) {
+            this.processRunner = processRunner;
+            this.processName = processName;
+            timer = new Timer(true);
+        }
+
+        @Override
+        public Integer call() throws IOException, InterruptedException {
+            timer.schedule(new PrintTask(), 0, 1000);
+            final int result = processRunner.runProcess(processName);
+            timer.cancel();
```

Comment:
when the process ends, print a new line, otherwise the next log message is printed in the same line after the dots.